tests/end2end/MeetingSeries_security.js

```diff
@@ -0,0 +1,100 @@
+import { E2EGlobal } from './helpers/E2EGlobal'
+import { E2EApp } from './helpers/E2EApp'
+import { E2ESecurity } from './helpers/E2ESecurity'
+import { E2EMeetingSeries } from './helpers/E2EMeetingSeries'
+
+describe('MeetingSeries Security', function () {
+    const insertMeetingSeriesMethod = "meetingseries.insert";
+    const removeMeetingSeriesMethod = "workflow.removeMeetingSeries";
+    const updateMeetingSeriesMethod ="meetingseries.update";
+
+    beforeEach("goto start page and make sure test user is logged out", function () {
+        E2EApp.gotoStartPage();
+        E2EApp.loginUser();
+        expect(E2EApp.isLoggedIn()).to.be.true;
+    });
+
+    before("reload page and reset app", function () {
+        E2EApp.resetMyApp(true);
+        E2EApp.launchApp();
+    });
+
+    it('can not insert a new MeetingSerie if not logged in', function () {
+        let aProjectName = "Hacker Project #1";
```

Comment:
Declare the variables as constants (e.g. `const aProjectName = "Hacker Project #1";`) to denote that it won't be modified. Please check the following variable declarations, too. Thanks!